app/services/search/location_builder.rb

```diff
@@ -2,6 +2,7 @@
 
 class Search::LocationBuilder
   DEFAULT_RADIUS = 10
+  NATIONWIDE_LOCATIONS = ["england", "uk", "united kingdom", "britain", "great britain"].freeze
```

Comment:
`%w[blah blah blah]`

Reply:
As we have "united kingdom" and "great britain", it can't be done it seems. It'd create separate elements for each word.

Reply:
Oh yeah good point lol